src/olympia/stats/tests/test_views.py

```diff
@@ -918,6 +920,234 @@ def test_downloads_sources_csv(self):
             2009-06-01,1000,550,450""",
         )
 
+    def test_download_by_content_json(self):
+        self.get_download_series_mock.return_value = [
```

Comment:
Most of this setup is repeated in each test. I.e., the `date`, `end` and `count` are identical in each test, with only the `data` changing. Do you see any value in creating a helper function that returns this, and thereby being able to remove a lot of code from each test. Or do you think it's better to have everything explicit in each test?

Reply:
I tried to refactor a bit but I could get a parametrized test working correctly. This is repetitive for sure but it is also very explicit.. Maybe some sort of helper would help (hehe) but I also like that we have everything in each test case.